backend/app/src/main/java/com/onair/hearit/explore/application/HearitExploreService.java

```diff
@@ -14,13 +14,17 @@
 public class HearitExploreService {
 
     private final List<ExploreScoreProcessor> exploreScoreProcessors;
+    private final ExploreScoreRefresher exploreScoreRefresher;
 
     public CursorResponseV2<ExploredHearitResponse> getExploredHearits(UserInfo userInfo,
                                                                        CursorRequest cursorRequest) {
         ExploreScoreProcessor exploreScoreProcessor = getExploreScoreProcessor(userInfo);
+        String userUuid = exploreScoreProcessor.resolveUserUuid(userInfo);
+        exploreScoreRefresher.refreshIfNeeded(cursorRequest.cursorId(), userUuid, userInfo.getUserType());
```

Comment:
```suggestion
        ExploreScoreProcessor exploreScoreProcessor = getExploreScoreProcessor(userInfo);
        exploreScoreProcessor.refreshIfNeeded(cursorRequest.cursorId(), userInfo);
```

refresher라는 클래스를 추가하기 보다 이렇게 하는거 어떤가요??
왜냐면 지금 exploreScoreProcessor에서 resolverUserUuid를 받아서 -> 다시 exploreScoreRefresher에 점수를 갱신하고 -> 다시 exploreScoreProcessor에서 응답을 계산하잖아요.

이렇게 중간에 exploreScoreProcessor가 중간에 낀 이유가 결국 member, guest랑 다르기때문인데 지금 exploreScoreRefresher는 그걸 모르게 하고 싶으니까 그렇게 된 것 같아요. 
차라리 guest, member를 아는 애들이 refresh하는 책임도 가지면 중간에 resolverUserUuid를 service에서 호출할 필요도 없고 흐름도 더 이해가 잘될 것 같아용

대신 벡터가 말한 외부에서 갱신하는걸 알아야한다는건 동의하기때문에 public으로 메서드를 선언해서 호출해주면 되지 않을까요?

Reply:
그럼 결국 ExploreScoreProcessor라는 클래스에 여러 책임이 혼재되게 된다는 점은 동일해지는 거 같음요
테스트할 때 ExploreScoreProcessor에 초기화로직까지 테스트해야된다는 점요

Reply:
저도 가콩이 말한 문제  때문에 고민이 생겨서 제안서 하나 더 만드려다가 말았었는데 processor로부터 아예 UserInfo를 받아 userUuid를 반환하는 클래스를 또 따로 분리하는 건 어떰요? 

결국 나중에 점수 갱신 로직은 스케줄러든, 어떤 식으로든 리팩토링 될 수 있을 것 같은데 같은데 이 경우 exploreScoreProcessor에 초기화와 조회 두 가지 책임이 혼재되어 있어 유지 보수가 힘들 것 같아서요

Reply:
논의 후 processor가 refresher를 가지도록 변경했습니다
https://github.com/woowacourse-teams/2025-hEARit/pull/615/commits/a9e2fb92d594883abb862010f4a39e69bd5c60ff